lib/rules/web/admin_console/4.1/deployment.xyaml

```diff
@@ -45,3 +45,11 @@ delete_dc_with_dependency_objects:
     button_text: Delete
   action: click_one_dropdown_action
   action: click_button
+check_sections_on_container_details:
```

Comment:
In 4.4, `Container Overview` title is changed to `Container Details`
<img width="1252" alt="Screen Shot 2020-02-11 at 4 52 49 PM" src="https://user-images.githubusercontent.com/12692381/74222270-13b8e300-4cef-11ea-8ae9-a6e96fea00ae.png">